### PR TITLE
unixPB: add new rise paid for scaleway RISC-V hosts

### PR DESCRIPTION
**Pull request by gdams**
This will replace the existing rise hosts

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] VPC/QPC not applicable for this PR
- [x] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Providers/tasks/main.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@
     - adoptopenjdk
 
 #####################
```
Comment (karianna) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Providers/tasks/main.yml:69]:
```suggestion
############################
```

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Providers/tasks/main.yml [hunk 2]
```diff
@@ -67,7 +67,7 @@
...
-# Marist & Scaleway #
+# Marist & Rise (Scaleway) #
 #####################
```
Comment (karianna) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Providers/tasks/main.yml:71]:
```suggestion
############################
```

**Review by karianna — APPROVED** [review]

**Review by steelhead31 — APPROVED** [review]
LGTM

**Review by sxa — COMMENTED** [review]
We should get these added to Bastillion before merging this (as per the last checkbox in the template). I've done the first one as a test.

**Review by karianna — APPROVED** [review]

**Pull request merged**